lib/node_modules/@stdlib/blas/base/sdsdot/src/sdsdotsub.f

```diff
@@ -18,13 +18,13 @@
 
 !> Wraps `sdsdot` as a subroutine.
 !
-! @param {integer} N - number of values over which to compute the dot product
+! @param {integer} N - number of values
 ! @param {real} sb - scalar constant added to the dot product
 ! @param {Array<real>} sx - first array
 ! @param {integer} strideX - `sx` stride length
 ! @param {Array<real>} sy - second array
 ! @param {integer} strideY - `sy` stride length
-! @param {real} dot - output variable reference
+! @param {real} dot - dot product
```

Comment:
```suggestion
! @param {real} dot - output variable reference
```

This can stay as is, as the previous line was more descriptive in terms of intent.